packages/OmegaPrint-Core.package/CodeHolder.extension/instance/showingOmegaPrint.st

```diff
@@ -0,0 +1,4 @@
+*OmegaPrint-Core
+showingOmegaPrint
```

Comment:
diese Methode gibt einen Boolean zurück: warum heißt sie nicht mehr **is**ShowingOmegaPrint?

Reply:
![image](https://user-images.githubusercontent.com/48409110/89445330-48b06a80-d753-11ea-8cb5-8ae5ec3eeb0a.png)
Keine der Methoden faengt mit `is` an, ich denke unsere sollte es dann auch nicht

Reply:
Hm nur weil es im Image nicht so ist, weiß nicht ob wir es dann auch "falsch" machen sollten, aber ist ein Argument

Reply:
genau das war damals unser Argument, weshalb wir es dennoch isShowing genannt hatten

Reply:
ich reverte das mal

Reply:
ist gefixt, bitte angucken und resolven

Reply:
dann mache ich das mal :D